api/src/main/java/io/opentelemetry/trace/BigendianEncoding.java

```diff
@@ -138,6 +138,15 @@ static void byteToBase16String(byte value, char[] dest, int destOffset) {
     byteToBase16(value, dest, destOffset);
   }
 
+  static byte[] bytesFromBase16(CharSequence value, int srcOffset, int charactersToRead) {
+    CharSequence part = value.subSequence(srcOffset, srcOffset + charactersToRead);
```

Comment:
Think the loop is still very easy even without the subsequence

Reply:
definitely. I thought I had caught all these, but I missed this one. will fix.

Reply:
done